.gitignore

```diff
@@ -0,0 +1,3 @@
+/data
+/out
```

Comment:
This works but it's a little non-standard. The more standard way would be:

```
data
out
```

or

```
data/
out/
```

The trailing slash is only necessary to indicate to readers of this file that they are directories.

Reply:
Updated